src/Hamiltonians/offdiagonals.jl

```diff
@@ -3,7 +3,7 @@
 ### operating on Bosonic addresses.
 ###
 """
-    AbstractOffdiagonals{A<:AbstractFockAddress,T}<:AbstractVector{Tuple{A,T}}
+    AbstractOffdiagonals{A,T}<:AbstractVector{Tuple{A,T}}
```

Comment:
I do not think that there is a good reason to expect all addresses to be subtype of `AbstractFockAddress`. We should allow users to define their own abstract types that have nothing to do with Fock space.

Reply:
Makes sense.